lib/private/AppFramework/Utility/SimpleContainer.php

```diff
@@ -52,6 +52,11 @@ public function __construct() {
 		$this->container = new Container();
 	}
 
+	/**
+	 * @template T
+	 * @param class-string<T> $id
```

Comment:
```suggestion
     * @psalm-suppress MoreSpecificImplementedParamType
	 * @param class-string<T> $id
```

This should hopefully fix the issue :)